app/controllers/application_controller.rb

```diff
@@ -188,4 +192,15 @@ def schedule_mode?
   def program_tracks
     @program_tracks ||= current_event && current_event.tracks.any? ? current_event.tracks : []
   end
+
+  def load_sponsors
+    @sponsors = Sponsor.published
```

Comment:
@cody This code is looking a bit suspicious to me. I suspect that the Ruby could be simplified and perhaps more of it could be done with ActiveRecord?

Reply:
Thanks for pointing this out!
Moved all of this logic into scopes on the Model. 

.order_by_tier - Arranges the sponsors in the TIER array order
.with_footer_image - Sponsors that have a footer image attached

Still used the group_by in the controller..